pandas/tests/indexing/test_check_indexer.py

```diff
@@ -31,15 +31,16 @@ def test_valid_input(indexer, expected):
     tm.assert_numpy_array_equal(result, expected)
 
 
-@pytest.mark.parametrize(
-    "indexer", [[True, False, None], pd.array([True, False, None], dtype="boolean")],
-)
-def test_bool_raise_missing_values(indexer):
-    array = np.array([1, 2, 3])
+def test_boolean_na_returns_indexer():
+    # TODO: What to do now about list input that contains
+    # bools and None?
```

Comment:
Test the behaviour, I would say (and I think it will convert a list to boolean array, so then further treats it as such)

Reply:
Yes, looks like it's behaving the same way as a boolean array with NA, but I wanted to note this because it might be unexpected to silently treat `None` as `pd.NA` when passed in a list. I'll add it back to the test, which should pass as it's written now.

Reply:
Ah, yes, that would indeed be a change in behaviour .. 
I *think* I am fine with that (it's a corner case anyway to pass such mask as a list), but not fully sure. We can also punt for now on the list case (at least for Series, on the array level it should already work)